enacts/wat_bal/agronomy.py

```diff
@@ -189,6 +241,15 @@ def soil_plant_water_balance(
     ).assign_coords({time_dim: peffective[time_dim][0] - np.timedelta64(1, "D")})
     # sm depends on sminit dims and time_dim
     sm = sminit.drop_vars(time_dim).broadcast_like(peffective[time_dim]) * np.nan
+    # et_crop_red depends on sm and rho_crop dims
+    if rho_crop is None:
+        ks = 1
+        et_crop_red = et_crop
+    else:
+        rho_crop = xr.DataArray(rho_crop)
+        if not rho_adj: # raw is constant against time
+            raw = rho_crop * taw
+        et_crop_red = sm.broadcast_like(rho_crop)
```

Comment:
I added some comments accross.

Reply:
"Initializing" means that you're providing initial values... presumably meaningful ones. I would prefer "allocating", since all you're really doing is making space to hold the data, not providing any meaningful data.